eng/common/scripts/get-codeowners.ps1

```diff
@@ -1,49 +1,50 @@
 param (
-  $TargetDirectory, # should be in relative form from root of repo. EG: sdk/servicebus
-  $RootDirectory, # ideally $(Build.SourcesDirectory)
-  $VsoVariable = "" # target devops output variable
+  [string]$CodeOwnerPathExpression, # Code path to code owners. e.g sdk/core/azure-amqp
```

Comment:
This isn't the path to code owners this should be called "relativePathInRepoToFindOwners" or something like that.